client/ayon_nuke/api/plugin.py

```diff
@@ -309,6 +311,24 @@ def set_selected_nodes(self, pre_create_data):
         else:
             self.selected_node = None
 
+    def update_instances(self, update_list):
+        for created_inst, changes in update_list:
+            instance_node = created_inst.transient_data["node"]
+
+            update_write_node_filepath(created_inst, changes)
+            # in case node is not existing anymore (user erased it manually)
+            try:
+                instance_node.fullName()
+            except ValueError:
+                self.remove_instances([created_inst])
+                continue
+
```

Comment:
```suggestion
            # in case node is not existing anymore (user erased it manually)
            try:
                instance_node.fullName()
            except ValueError:
                self.remove_instances([created_inst])
                continue
            
            update_write_node_filepath(created_inst, changes)
```
I suppose we should first check for the deleted node?